python-mini-projects/projects/Calculate_age/calculate.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import time
+from calendar import isleap
+
+# judge the leap year
+def judge_leap_year(year):
+    if isleap(year):
+        return True
+    else:
+        return False
+
+
+# returns the number of days in each month
+def month_days(month, leap_year):
+    if month in [1, 3, 5, 7, 8, 10, 12]:
+        return 31
+    elif month in [4, 6, 9, 11]:
+        return 30
+    elif month == 2 and leap_year:
+        return 29
+    elif month == 2 and (not leap_year):
+        return 28
+
+
+name = input("input your name: ")
+age = input("input your age: ")
+localtime = time.localtime(time.time())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Use the `datetime` module for time zone conversions. The older `time` module doesn&#39;t work consistently with multiple time zones and shouldn&#39;t be used.

[Learn more](https://docs.python.org/3/library/time.html#module-time.)